audio/beets/Portfile

```diff
@@ -85,13 +85,20 @@ if {${name} eq ${subport} || "${name}-devel" eq ${subport}} {
                     port:py${python.version}-pyxdg \
                     port:py${python.version}-yaml
 
+    # ffmpeg-devel now tracks ffmpeg7,
+    # so no path-style for ffmpeg v. 4.x.
     depends_run-append \
                     port:chromaprint \
-                    path:lib/libavcodec.dylib:ffmpeg \
-                    port:flac \
-                    port:mp3val \
                     port:ImageMagick \
+                    port:ffmpeg \
+                    port:flac \
+                    port:mp3val
+
+    # Do not pull in Go on PowerPC
+    if {${os.arch} ne "powerpc"} {
+        depends_run-append \
                     port:ipfs
+    }
```

Comment:
You are changing runtime/library dependencies here, so please increase the "revision".

Reply:
Done